operator/data/profiles/demo.yaml

```diff
@@ -105,12 +104,5 @@ spec:
       istio-ingressgateway:
         autoscaleEnabled: false
 
-    grafana:
-      enabled: true
```

Comment:
??

Reply:
this part is already defined in the addonComponents part above